### PR TITLE
page props types updated with IResourceComponentProps

### PR DESCRIPTION
**Pull request by ozkalai**
Test me! 'MUI'
[Link to PAGES-PROPS-TYPE-FIX](https://pages-p-refine.pankod.com)

In this Pull Request
The pages prop have been updated with IResourceComponentProps in the fine food example

### CODE REVIEW

**Review by biskuvit — APPROVED** [review]

File: examples/fineFoods/admin/mui/src/pages/couriers/show.tsx [hunk 1]
```diff
@@ -40,21 +41,28 @@ export const CourierShow: React.FC = () => {
...
-                gap={1}
-            >
-                {icon}
-                <Typography variant="body1">{text}</Typography>
-            </Stack>
-        );
+    type CourierInfoTextProps = IResourceComponentsProps & {
+        icon: React.ReactNode;
+        text?: string;
+    };
```
Comment (salihozdemir) [inline examples/fineFoods/admin/mui/src/pages/couriers/show.tsx:47]: You should pass the `IResourceComponentsProps` to the resources pages that are given the resources in App.tsx. In this page you should have passed it to the `CourierShow` component like this:

`export const CourierShow: React.FC<IResourceComponentsProps> = () => {`

File: examples/fineFoods/admin/mui/src/pages/dashboard/index.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ import {
     TrendingMenu,
 } from "components/dashboard";
 
-export const DashboardPage: React.FC = () => {
+export const DashboardPage: React.FC<IResourceComponentsProps> = () => {
```
Comment (salihozdemir) [inline examples/fineFoods/admin/mui/src/pages/dashboard/index.tsx:15]: Does `DashboardPage` have not any props, doesn't it? @omeraplak  

So, you don't need to pass `IResourceComponentsProps` to the dashboard page.

File: examples/fineFoods/admin/mui/src/pages/login/index.tsx [hunk 3]
```diff
@@ -26,7 +27,7 @@ type ILoginForm = {
     remember?: boolean;
 };
 
-export const LoginPage: React.FC = () => {
+export const LoginPage: React.FC<IResourceComponentsProps> = () => {
```
Comment (salihozdemir) [inline examples/fineFoods/admin/mui/src/pages/login/index.tsx:30]: Same thing, `LoginPage` doesn't have any default props.

**Review by salihozdemir — CHANGES_REQUESTED** [review]

File: examples/fineFoods/admin/mui/src/pages/dashboard/index.tsx (hunk 2 above)
Comment (omeraplak) [reply examples/fineFoods/admin/mui/src/pages/dashboard/index.tsx:15]: @salihozdemir yes, you are right. @ozkalai We don't need this time (THANK YOU MARIO! BUT OUR PRINCESS IS IN ANOTHER CASTLE! 🤣)

**Review by omeraplak — APPROVED** [review]

**Pull request merged**